app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,24 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
True enough - I like the root font size at 10px, so it make sense to me to specify 10px explicitly. I've updated accordingly!